src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -41,11 +43,28 @@
         };
     }
 
+    public static Task[] getSampleTasks() {
```

Comment:
Might want to add some sample tasks with assigned people for better testing

Reply:
Done!